sources/constants/README.md

```diff
@@ -0,0 +1,9 @@
+# constants
+
+Current version: 0.1.0
+
+  This crate contains constants shared accross multiple bottlerocket's crates
```

Comment:
Across is misspelled :)

I think this should probably read `shared across multiple Bottlerocket crates`